apps/web/src/app/layout.tsx

```diff
@@ -51,7 +52,7 @@ export const metadata = {
   },
 };
 
-export default function RootLayout({ children }: { children: React.ReactNode }) {
+export default async function RootLayout({ children }: { children: React.ReactNode }) {
```

Comment:
any reason for the change? based on the content below I'm not sure this is needed.

Reply:
sorry for the inconvinience, i agree, it shouldn't be there